CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## Current (in progress)
+
+- Add bumpr to manage changelog [#23](https://github.com/opendatateam/udata-search-service/pull/23)
```

Comment:
Since it's going to be the only entry (almost) in the changelog, can we add something more generic ?
like `Initial version of udata-search-service` ?